src/main/kotlin/org/rust/cargo/util/CargoArgsSeparator.kt

```diff
@@ -0,0 +1,123 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.cargo.util
+
+class CargoRunArgsSeparator(cargoArgs: List<String>) : AbstractCargoArgsSeparator(cargoArgs, RUN_OPTIONS) {
+    override val commandOptions: List<String> by lazy {
+        separatedArgs.first
+    }
+
+    override val executableArguments: List<String> by lazy {
+        val positionalArguments = separatedArgs.second
```

Comment:
Hm, is it right name? Looks like it should be `executableArguments`

Reply:
This is the correct name, but I chose the wrong name for `executableArguments` in the `separatedArgs` property.

Now I use the following names:
* `commandOptions` -- options along with their arguments
* `positionalArguments` -- all positional arguments
* `commandArguments` -- `commandOptions` + part of `positionalArguments`
* `executableArguments` -- rest of `positionalArguments`